### DISCUSSION TITLE
Audio subtitles cannot be aligned with timeline

### DISCUSSION THREAD

**Discussion by machenme** [openai/whisper#1995] [Q&A] [answered]
Hello everyone. I tried to use whisper to extract subtitles from mp3 audio, but I found that the start and end times did not match the actual time of the audio. What should I do
![image](https://github.com/openai/whisper/assets/117664524/8b530acf-d3c0-4b75-83f6-8253437ed9b8)

**Comment by ryanheise** [comment, marked as answer]
You may try running with the `--word_timestamps True` option to get more accurate timestamps. Although this option adds word timestamps, those word timestamps are also used to improve the accuracy of the segment timestamps.

**Reply by machenme** [reply]
here is what i add `word_timestamps=True`, you help me a lot. thank you
![image](https://github.com/openai/whisper/assets/117664524/c49b1762-7d53-4952-b142-059ad6402ffc)

**Comment by toanhuynhnguyen** [comment]
Try WhisperX
